From 7d28712eff3ee8a7d3d57a9631b52f148078a962 Mon Sep 17 00:00:00 2001
From: Patrick Rudolph <patrick.rudolph@9elements.com>
Date: Tue, 24 Oct 2023 08:40:34 +0200
Subject: [PATCH 01/15] configurations: sbp1: Swap PVCCIN and PVCCFA_EHV_FIRA

PVCCIN and PVCCFA_EHV_FIRA are on the same VR, but the channels
got mixed up. PVCCIN is actually the first channel and
PVCCFA_EHV_FIRA the second. Also update name from FIRA to FIVRA.

Change-Id: I3eded98f9ea85b35d925b574b091492c58017bb8
Signed-off-by: Patrick Rudolph <patrick.rudolph@9elements.com>
---
 configurations/sbp1_baseboard.json | 48 +++++++++++++++---------------
 1 file changed, 24 insertions(+), 24 deletions(-)

diff --git a/configurations/sbp1_baseboard.json b/configurations/sbp1_baseboard.json
index e97bf91..9bfd28e 100644
--- a/configurations/sbp1_baseboard.json
+++ b/configurations/sbp1_baseboard.json
@@ -1627,8 +1627,8 @@
                 "pout2",
                 "temp1"
             ],
-            "Name": "PVCCFA_EHV_FIRA_CPU0",
-            "Name1": "PVCCIN_CPU0",
+            "Name": "PVCCIN_CPU0",
+            "Name1": "PVCCFA_EHV_FIVRA_CPU0",
             "PollRate": 5.0,
             "PowerState": "Always",
             "Thresholds": [
@@ -1665,14 +1665,14 @@
                     "Label": "iout1",
                     "Name": "upper critical",
                     "Severity": 1,
-                    "Value": 164
+                    "Value": 634.0
                 },
                 {
                     "Direction": "greater than",
                     "Label": "iout1",
                     "Name": "upper non critical",
                     "Severity": 0,
-                    "Value": 102
+                    "Value": 550.0
                 },
                 {
                     "Direction": "greater than",
@@ -1693,14 +1693,14 @@
                     "Label": "iout2",
                     "Name": "upper critical",
                     "Severity": 1,
-                    "Value": 550.0
+                    "Value": 164.0
                 },
                 {
                     "Direction": "greater than",
                     "Label": "iout2",
                     "Name": "upper non critical",
                     "Severity": 0,
-                    "Value": 634.0
+                    "Value": 102.0
                 }
             ],
             "Type": "MP2973"
@@ -1720,8 +1720,8 @@
                 "pout2",
                 "temp1"
             ],
-            "Name": "PVCCFA_EHV_FIRA_CPU1",
-            "Name1": "PVCCIN_CPU1",
+            "Name": "PVCCIN_CPU1",
+            "Name1": "PVCCFA_EHV_FIVRA_CPU1",
             "PollRate": 5.0,
             "PowerState": "Always",
             "Thresholds": [
@@ -1758,14 +1758,14 @@
                     "Label": "iout1",
                     "Name": "upper critical",
                     "Severity": 1,
-                    "Value": 164
+                    "Value": 634.0
                 },
                 {
                     "Direction": "greater than",
                     "Label": "iout1",
                     "Name": "upper non critical",
                     "Severity": 0,
-                    "Value": 102
+                    "Value": 550.0
                 },
                 {
                     "Direction": "greater than",
@@ -1786,14 +1786,14 @@
                     "Label": "iout2",
                     "Name": "upper critical",
                     "Severity": 1,
-                    "Value": 550.0
+                    "Value": 164.0
                 },
                 {
                     "Direction": "greater than",
                     "Label": "iout2",
                     "Name": "upper non critical",
                     "Severity": 0,
-                    "Value": 634.0
+                    "Value": 102.0
                 }
             ],
             "Type": "MP2973"
@@ -1813,8 +1813,8 @@
                 "pout2",
                 "temp1"
             ],
-            "Name": "PVCCFA_EHV_FIRA_CPU2",
-            "Name1": "PVCCIN_CPU2",
+            "Name": "PVCCIN_CPU2",
+            "Name1": "PVCCFA_EHV_FIVRA_CPU2",
             "PollRate": 5.0,
             "PowerState": "Always",
             "Thresholds": [
@@ -1851,14 +1851,14 @@
                     "Label": "iout1",
                     "Name": "upper critical",
                     "Severity": 1,
-                    "Value": 164
+                    "Value": 634.0
                 },
                 {
                     "Direction": "greater than",
                     "Label": "iout1",
                     "Name": "upper non critical",
                     "Severity": 0,
-                    "Value": 102
+                    "Value": 550.0
                 },
                 {
                     "Direction": "greater than",
@@ -1879,14 +1879,14 @@
                     "Label": "iout2",
                     "Name": "upper critical",
                     "Severity": 1,
-                    "Value": 550.0
+                    "Value": 164.0
                 },
                 {
                     "Direction": "greater than",
                     "Label": "iout2",
                     "Name": "upper non critical",
                     "Severity": 0,
-                    "Value": 634.0
+                    "Value": 102.0
                 }
             ],
             "Type": "MP2973"
@@ -1906,8 +1906,8 @@
                 "pout2",
                 "temp1"
             ],
-            "Name": "PVCCFA_EHV_FIRA_CPU3",
-            "Name1": "PVCCIN_CPU3",
+            "Name": "PVCCIN_CPU3",
+            "Name1": "PVCCFA_EHV_FIVRA_CPU3",
             "PollRate": 5.0,
             "PowerState": "Always",
             "Thresholds": [
@@ -1944,14 +1944,14 @@
                     "Label": "iout1",
                     "Name": "upper critical",
                     "Severity": 1,
-                    "Value": 164
+                    "Value": 634.0
                 },
                 {
                     "Direction": "greater than",
                     "Label": "iout1",
                     "Name": "upper non critical",
                     "Severity": 0,
-                    "Value": 102
+                    "Value": 550.0
                 },
                 {
                     "Direction": "greater than",
@@ -1972,14 +1972,14 @@
                     "Label": "iout2",
                     "Name": "upper critical",
                     "Severity": 1,
-                    "Value": 550.0
+                    "Value": 164.0
                 },
                 {
                     "Direction": "greater than",
                     "Label": "iout2",
                     "Name": "upper non critical",
                     "Severity": 0,
-                    "Value": 634.0
+                    "Value": 102.0
                 }
             ],
             "Type": "MP2973"
-- 
2.41.0

